db/all-ideas/app-ideas/randomactivity.mdx

```diff
@@ -0,0 +1,16 @@
+--- 
+title: Create a Activity Suggestion App
+slug: randomactivity
+createdAt: Sun Oct 16 2022 23:41:57 GMT+0900 (Korean Standard Time)
+category: app-ideas
+summary: Activity suggestion app help bored users decide on what they want to randomly do on that day.
+tags: javascript, css
+author: LeGrandMAG
+---
+
+# Create a Activity Suggestion App
```

Comment:
😞

Reply:
Your instructions don't mention anything about the body of the markdown file.

When I run the app locally, it prompts me about the title, the author, the smug, the tags, and the summary, but it does not say anything about the body. I think there should be some more instructions regarding completing the body of the markdown file, or when the user runs plop, let them know that they should complete the body (description) of their app manually. 

I created an issue about that here #20